desktop/apps/auction2/templates/index.jade

```diff
@@ -13,3 +12,5 @@ block body
 
       .auction2-my-active-bids
       #cf-artworks
+      if articles && articles.length > 0 || auction.get('eligible_sale_artworks_count') === 0
```

Comment:
Here's the logic for determining whether or not we show the footer-- basically, only if there are articles or if the auction has no artworks. For the standard use-case (many artworks, infinite scroll), we don't want to show it.